modules/collect/src/main/java/com/opengamma/strata/collect/result/FailureItem.java

```diff
@@ -78,8 +85,8 @@
   /**
    * Obtains a failure from a reason and message.
    * <p>
-   * The message is produced using a template that contains zero to many "{}" placeholders.
-   * Each placeholder is replaced by the next available argument.
+   * The message is produced using a template that contains zero to many "{}" or "{abc}" placeholders.
+   * Each placeholder is replaced by the next available argument. If the placeholder is name, it will be added to the attributes map.
```

Comment:
Typo "is name" should be "is named" or "has a name". Also the rest of the sentence could be clearer. Maybe something like "it's value is added to the attribute map with the name as the key".